mindmeld/models/helpers.py

```diff
@@ -463,3 +466,60 @@ def add_resource(func):
         return func
 
     return add_resource
+
+
+class FileBackedList:
+    """
+    FileBackedList implements an interface for simple list use cases
+    that is backed by a temporary file on disk.  This is useful for
+    simple list processing in a memory efficient way.
+    """
```

Comment:
Why couldnt we use `https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryFile` instead of implementing our own version?